lib/etso/adapter.ex

```diff
@@ -51,8 +53,19 @@ defmodule Etso.Adapter do
   @impl Ecto.Adapter
   def dumpers(:binary_id, type), do: [type, Ecto.UUID]
   def dumpers(:embed_id, type), do: [type, Ecto.UUID]
+  def dumpers(:map, type), do: [type, Etso.Ecto.MapType]
   def dumpers(_, type), do: [type]
 
+  @doc """
+  Delete all data from the tables.
+  """
+  @spec flush_tables(module()) :: :ok
+  def flush_tables(repo) do
```

Comment:
I would think the better approach would be to write a helper function in each application, which deletes everything from every table explicitly, as no other adapters have this function

Reply:
ok. I can take this out.